### PR TITLE
Converting a couple more functional tests to "fast"

### PR DESCRIPTION
**Pull request by rjrudin**
Also starting to convert duplicated code in ConnectedRESTQA to use ml-app-deployer

### CODE REVIEW

File: marklogic-client-api-functionaltests/src/test/java/com/marklogic/client/fastfunctest/ClientApiFunctionalTest.java [hunk 1]
```diff
@@ -1,1180 +1,1003 @@
...
-
-	/**
-	 * Invokes the xqyfunction operation on the database server
-	 *
-	 * @param items	provides input
-	 * @return	as output
-	 */
-	String xqyfunction(String items);
-
+package com.marklogic.client.fastfunctest;
```
Comment (rjrudin) [inline marklogic-client-api-functionaltests/src/test/java/com/marklogic/client/fastfunctest/ClientApiFunctionalTest.java:1]: Think I must have hit "format code" on this file, which is why it all appears as a diff. Sigh.

File: marklogic-client-api-functionaltests/src/test/java/com/marklogic/client/functionaltest/ConnectedRESTQA.java [hunk 2]
```diff
@@ -1018,101 +1022,18 @@ public static void tearDownJavaRESTServer(String dbName, String[] fNames, String
...
-				System.out.println("REST call for database properties update has issues");
-				System.out.println(responsePut.toString());
-			}
-		} catch (Exception e) {
-			e.printStackTrace();
-		} finally {
-			client = null;
-		}
+	public static void setDatabaseProperties(String dbName, String prop, String propValue) {
+		ObjectNode properties = new ObjectMapper().createObjectNode();
```
Comment (rjrudin) [inline marklogic-client-api-functionaltests/src/test/java/com/marklogic/client/functionaltest/ConnectedRESTQA.java:1026]: ConnectedRESTQA has a lot of duplicated code on the left for making calls to the Manage API. Going to gradually convert those all over to use ManageClient from the ml-app-deployer. Will get rid of hundreds of lines of duplicated code.

**Review by BillFarber — APPROVED** [review]
I'm just assuming ClientApiFunctionalTest is correct.

**Comment by rjrudin** [comment]
Yup, ClientApiFunctionalTest is still passing. I converted it to an AbstractFunctionalTest and also removed a few dozen warnings that Intellij was reporting.

**Pull request merged**